app/screens/RestaurantDetail/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { React, useEffect, useState } from 'react';
+import {
+  ScrollView, Text, StyleSheet, FlatList,
+} from 'react-native';
+import axios from 'axios';
+import { Card, Rating, AirbnbRating } from 'react-native-elements';
+import {
+  PhoneIcon as PhoneIconOutline, MapPinIcon as MapPinIconOutline, ClockIcon as ClockIconOutline, ArrowSmallRightIcon as ArrowSmallRightIconOutline, TruckIcon as TruckIconOutline, BookmarkIcon as BookmarkIconOutline,
+} from 'react-native-heroicons/outline';
+import ImgCarousel from './ImgCarousel';
+import Reviews from './Reviews';
+
+const styles = StyleSheet.create({
+  icons: {
+    width: 16,
+    heigth: 16,
+    color: '#55d6bf',
+  },
+  bookmarkIcon: {
+    width: 20,
+    heigth: 20,
+  },
+});
+
+function RestaurantDetail({ route, navigation }) {
+  const item = route.params.item;
+  const [detail, setDetail] = useState({});
+
+  useEffect(() => {
+    axios.get(`http://localhost:8080/restaurants/${item.APIID}`)
```

Comment:
Use a query string with the yelpId if still needed after my PR merges instead of APIID. The id of the restaurants endpoint refers to the primary key of the restaurants in the database.